package.json

```diff
@@ -2,9 +2,10 @@
   "name": "@airgap/beacon-sdk",
   "version": "1.1.0",
   "description": "The beacon-sdk allows you to easily connect DApps with Wallets through P2P communication or a chrome extension.",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "unpkg": "dist/beacon.min.js",
-  "types": "dist/index.d.ts",
+  "types": "dist/cjs/index.d.ts",
```

Comment:
I'm not sure if this matters, but both ESM and CJS will read types from the `dist/cjs` build. It looks like TypeScript type definition files are agnostic towards whatever module format they're describing, but I could be wrong here.